### ISSUE TITLE
Option to disable cyclic switching between Home and All categories.

### ISSUE THREAD

**Issue by tompinkas** [HenriDellal/emerald#32]
Right now the home button switches between the Home category and All apps. In my case, it's always goes first to All and only after second push of home button to Home. It would be nice, if there was an option, which disable this cyclic switching between two categories and let home button just go to Home category.

Similar as #28, perhaps this could be merged into single option...
